### PR TITLE
Prepare release notes and docs for AT 14.0

### PR DESCRIPTION
**Pull request by er-1**

### CODE REVIEW

File: configs/14.0/release_notes/release_notes-inst.html [hunk 1]
```diff
@@ -289,6 +289,39 @@ <h3>Using aptitude or apt</h3>
...
 
+<!-- if_ubuntu-20
+                        <p>The gpg public key <span class="code">xxxxxxxx-gpg.key</span>will be provided in the repository where these release notes were found.  The public key can be used to verify the authenticity of both the Advance Toolchain install packages and the repository contents. After downloading the gpg public key, it can be imported as follows:</p>
+                        <p class="code">sudo apt-key add xxxxxxxx.gpg.key</p>
+                        <h3>Using aptitude or apt</h3>
+                                <p>If the Advance Toolchain repository has not already been configured, that must be done first.  As <i>root</i>, edit the following file, adding one of the lines as needed for the installed system:
+                                <p class="code">/etc/apt/sources.list</p>
+                                <ul>
+                                        <li>For amd64 workstations or POWER servers (ppc64le) add the line:</li>
+                                        <p class="code">deb ftp://public.dhe.ibm.com/software/server/POWER/Linux/toolchain/at/ubuntu focal __VERSION__ </p>
```
Comment (tuliom) [inline configs/14.0/release_notes/release_notes-inst.html]: I've recently seen an error message on Ubuntu 20.04 complaining that FTP was not supported anymore.

File: configs/14.0/release_notes/relfixes.html [hunk 2]
```diff
@@ -11,6 +11,72 @@ <h3>Advance Toolchain new location</h3>
...
+  <ul>
+    <li>Version 1.1.1g, see the <a href="https://www.openssl.org/news/openssl-1.1.1-notes.html">release notes</a> for details.</li>
+  </ul>
+  <li>Expat provides:</li>
+  <ul>
+    <li>Version 2.2.9, see the <a href="https://github.com/libexpat/libexpat/blob/R_2_2_9/expat/Changes">release notes</a> for details.</li>
+  </ul>
+  <li>Libhugetlbfs provides:</li>
+  <ul>
+    <li>Version 2.21, see the <a href="https://github.com/libhugetlbfs/libhugetlbfs/blob/2.22/NEWS">release notes</a> for details.</li>
```
Comment (tuliom) [inline configs/14.0/release_notes/relfixes.html]: The NEWS file doesn't match the version listed.

File: configs/14.0/release_notes/relfixes.html [hunk 3]
```diff
@@ -11,6 +11,72 @@ <h3>Advance Toolchain new location</h3>
...
+  <ul>
+    <li>Version 1.73.0, see the <a href="https://www.boost.org/users/history/version_1_73_0.html">release notes</a> for details.</li>
+  </ul>
+  <li>libdfp provides:</li>
+  <ul>
+    <li>Version 0.15.0, see the <a href="https://github.com/libdfp/libdfp/releases/tag/1.0.15">release notes</a> for details.</li>
+  </ul>
+  <li>Valgrind provides:</li>
+  <ul>
+    <li>Version 3.15.0, see the <a href="http://valgrind.org/docs/manual/dist.news.html">release notes</a> for details.</li>
```
Comment (tuliom) [inline configs/14.0/release_notes/relfixes.html]: Valgrind 3.16.1 has already been released, but a tag wasn't created.
We may want to update this later.

Comment (er-1) [reply configs/14.0/release_notes/relfixes.html]: I missed the release end of June.
I agree to update Valgrind in AT 14.0-1.

File: configs/14.0/release_notes/release_notes-inst.html (hunk 1 above)
Comment (er-1) [reply configs/14.0/release_notes/release_notes-inst.html]: Fixed

File: configs/14.0/release_notes/relfixes.html (hunk 2 above)
Comment (er-1) [reply configs/14.0/release_notes/relfixes.html]: Fixed

**Comment by er-1** [comment]
> I think this is also missing references to POWER10 enablement and POWER10-optimized libraries (after merging PR #1589 )

I was waiting for the merge of #1589, but it's about to happen, so I added POWER10 lines.

File: configs/14.0/release_notes/relfixes.html [hunk 4]
```diff
@@ -1,16 +1,83 @@
+<li>POWER10 enablement.</li>
 <li>POWER9 enablement.</li>
 <li>POWER9 optimized scheduler.</li>
 <li>Requires at least POWER8 for ppc64le.</li>
 <li>POWER8 Transactional Memory enablement.</li>
 <li>POWER8 Crypto Operations enablement.</li>
-<li>POWER8 and POWER9 optimized system libraries.</li>
+<li>POWER8, POWER9 and POWER10 optimized system libraries.</li>
```
Comment (ThinkOpenly) [inline configs/14.0/release_notes/relfixes.html]: There's a debate about the "oxford comma", but my preference is to add a comma after "POWER9" here.

File: configs/14.0/release_notes/relfixes.html [hunk 5]
```diff
@@ -1,16 +1,83 @@
...
 <p>Since the end of August 2019, Advance Toolchain has been available from a new FTP server: <a href="ftp://public.dhe.ibm.com/">public.dhe.ibm.com</a></p>
 <p>Please be advised the FTP server at Unicamp won't be updated any longer.</p>
 
+<h3>New features in 14.0-0</h3>
+<ul>
+  <li>Support for Ubuntu 20.04.</li>
+  <li>GCC provides:</li>
+  <ul>
+    <li>Version 10.2, see the <a href="https://gcc.gnu.org/gcc-10/changes.html">release notes</a> for details.</li>
+    <li>MMA support.</li>
```
Comment (ThinkOpenly) [inline configs/14.0/release_notes/relfixes.html]: Do we want to provide some context for "MMA"? Is there a link for more information?

File: docs/advtool-installation.html [hunk 6]
```diff
@@ -62,109 +62,118 @@ <h4 class="ibm-h4">Taking advantage of leading edge tools and libraries</h4>
   <thead>
     <tr>
       <th></th>
+      <th>Advance Toolchain 14.0</th>
       <th>Advance Toolchain 13.0</th>
       <th>Advance Toolchain 12.0</th>
-      <th>Advance Toolchain 11.0</th>
```
Comment (ThinkOpenly) [inline docs/advtool-installation.html:67]: Are we dropping AT11 support?  If so, that might be nice to add to the commit message? Or, in a separate commit? Not a big deal, really.

File: configs/14.0/release_notes/relfixes.html (hunk 4 above)
Comment (er-1) [reply configs/14.0/release_notes/relfixes.html]: Done

File: configs/14.0/release_notes/relfixes.html (hunk 5 above)
Comment (er-1) [reply configs/14.0/release_notes/relfixes.html]: Added a link to GCC documentation.

File: docs/advtool-installation.html (hunk 6 above)
Comment (er-1) [reply docs/advtool-installation.html:67]: > Are we dropping AT11 support? 

We are.

> If so, that might be nice to add to the commit message? Or, in a separate commit?

You're right, much more cleaner.
I update the PR, a separate commit updates the documentation to remove AT 11.0.

File: configs/14.0/release_notes/relfixes.html [hunk 7]
```diff
@@ -1,16 +1,83 @@
...
+  </ul>
+  <li>libdfp provides:</li>
+  <ul>
+    <li>Version 0.15.0, see the <a href="https://github.com/libdfp/libdfp/releases/tag/1.0.15">release notes</a> for details.</li>
+  </ul>
+  <li>Valgrind provides:</li>
+  <ul>
+    <li>Version 3.15.0, see the <a href="http://valgrind.org/docs/manual/dist.news.html">release notes</a> for details.</li>
+  </ul>
+  <li>TCMmalloc provides:</li>
```
Comment (mscastanho) [inline configs/14.0/release_notes/relfixes.html]: Shouldn't this be `TCMalloc` instead?

File: configs/14.0/release_notes/relfixes.html [hunk 8]
```diff
@@ -1,16 +1,83 @@
...
+  <ul>
+    <li>Version 2.8.0, see the <a href="https://github.com/gperftools/gperftools/releases/tag/gperftools-2.8">release notes</a> for details.</li>
+  </ul>
+  <li>TBB provides:</li>
+  <ul>
+    <li>Version 2020.3, see the <a href="https://github.com/oneapi-src/oneTBB/blob/v2020.3/CHANGES">release notes</a> for details.</li>
+  </ul>
+  <li>Provides a new component: NX zlib compliant software (libnxz).</li>
+</ul>
+
```
Comment (mscastanho) [inline configs/14.0/release_notes/relfixes.html]: We should also add the zlib optimizations introduced by #1341 / #1547 to this list.

File: configs/14.0/release_notes/relfixes.html (hunk 7 above)
Comment (er-1) [reply configs/14.0/release_notes/relfixes.html]: Fixed

File: configs/14.0/release_notes/relfixes.html (hunk 8 above)
Comment (er-1) [reply configs/14.0/release_notes/relfixes.html]: Added a line about zlib.

**Pull request merged**